x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/attach_index_box.tsx

```diff
@@ -117,14 +119,17 @@ export const AttachIndexBox: React.FC<AttachIndexBoxProps> = ({ connector }) =>
                 ? ''
                 : i18n.translate(
                     'xpack.enterpriseSearch.attachIndexBox.euiFormRow.associatedIndexHelpTextLabel',
-                    { defaultMessage: 'You can use an existing index or create a new one' }
+                    { defaultMessage: 'You can use an existing index or create a new one.' }
                   )
             }
             error={
               showError
                 ? i18n.translate(
                     'xpack.enterpriseSearch.attachIndexBox.euiFormRow.associatedIndexErrorTextLabel',
-                    { defaultMessage: 'You can use another existing index or create a new one' }
+                    {
+                      defaultMessage:
+                        "Choose an index or create a new one. We can't create one with same name as it already exists.",
```

Comment:
```suggestion
                        "You can't create a new index using an existing index name. Choose an existing index or create a new index with a new name.",
```

Reply:
It makes much more sense to me! Thanks for your help, as always 🎩